client/my-sites/site-settings/settings-section-header.tsx

```diff
@@ -1,40 +1,38 @@
 import { Button } from '@automattic/components';
 import { useTranslate } from 'i18n-calypso';
-import PropTypes from 'prop-types';
+import { FC, PropsWithChildren, ReactNode } from 'react';
 import SectionHeader from 'calypso/components/section-header';
 
-const SettingsSectionHeader = ( {
+interface SettingsSectionHeaderProps extends PropsWithChildren {
+	title: string | ReactNode;
+	id?: string;
+	isSaving?: boolean;
+	showButton?: boolean;
+	disabled?: boolean;
+	onButtonClick?: () => void;
+}
+
+const SettingsSectionHeader: FC< SettingsSectionHeaderProps > = ( {
```

Comment:
FC is not necessary and largely in disuse these days. Components are simply functions, so let's type the props argument directly (see suggestion below).

```suggestion
function SettingsSectionHeader( {
```